### DISCUSSION TITLE
Unable to connect

### DISCUSSION THREAD

**Discussion by Cirion75** [vale46n1/immich_duplicate_finder#15] [Q&A]
I'm unable to connect to my immich:
![image](https://github.com/vale46n1/immich_duplicate_finder/assets/1172168/1af4c33b-0081-4897-bda8-6325d43921fb)

Same error on the external address.

What am I doing wrong?

**Comment by vale46n1** [comment]
> I'm unable to connect to my immich:
> ![image](https://github.com/vale46n1/immich_duplicate_finder/assets/1172168/9bc31c83-6fe3-4aa8-9109-93f7f4e35c57)
> 
> Same error on the external address.
> 
> What am I doing wrong?

I would suggest to cancel this image since you have exposed to anyone your server....don't share address and API.
Anyhow you should set the timeout to 2000ms.
Problem will be solved...I will add something to avoid such kind of situation

**Reply by Cirion75** [reply]
After setting the timeout to 2000ms I get the same message. I tried setting 2000000 with same message.
![image](https://github.com/vale46n1/immich_duplicate_finder/assets/1172168/d82e916d-0d05-4a57-b3a2-39299e335533)

**Comment by vale46n1** [comment]
Did you set the port on your address?

________________________________

**Reply by Cirion75** [reply]
I tried writing address like this:
http://ip:port
ip:port
https://domain.name
domain.name
https://domain.name:443
domain.name:443

And used fresh API key.

When using http:// or https:// in front it works longer with this:
![image](https://github.com/vale46n1/immich_duplicate_finder/assets/1172168/4b7d76e9-3d5c-4a7e-863f-3da6a3f6b4bb)

Then the same message.
